services/dy-mapcore-widget/Dockerfile

```diff
@@ -45,15 +45,16 @@ RUN  apt -y install nodejs
 # - build/compile
 # - replace the first slash from href=/ and src=/ in index.html
 
-RUN git clone --depth 1 https://github.com/ABI-Software/mapintegratedvuer.git mapintegratedvuer &&\
+RUN git clone https://github.com/alan-wu/mapintegratedvuer.git mapintegratedvuer &&\
     cd mapintegratedvuer &&\
-    git reset --hard b155ae2e151b607b149f9c4aa6a5825469cd801b &&\
+    git checkout 13-Apr-Simcore &&\
+    git reset --hard 638edaa37f8743300f1253fcd347b72a3a1e6a09 &&\
     npm install --save &&\
-    sed -i '/NavBar/d' src/components/MapIntegratedVuer.vue &&\
-    ./node_modules/.bin/vue-cli-service build &&\
-    sed -i s%href=/%href=%g dist/index.html &&\
-    sed -i s%src=/%src=%g dist/index.html
-
+    sed -i "s/openAnalyzer: false/openAnalyzer: false, analyzerMode: 'disabled'/" vue.config.js &&\
+    # sed -i '/NavBar/d' src/components/MapIntegratedVuer.vue &&\
+    ./node_modules/.bin/vue-cli-service build --mode static &&\
+    sed -i 's%href="/%href="%g' dist/index.html &&\
+    sed -i 's%src="/%src="%g' dist/index.html
```

Comment:
I guess these are the ne changes needed to properly run this version of the service